.github/workflows/jira-issue-create-template.yml

```diff
@@ -0,0 +1,69 @@
+# Template for jira issue creation in order to creates jira tickets for new github issues to help triage
+# Notice that the change of this temple might effect jira issue creation for other repositories
+name: Jira Issue Creator
```

Comment:
Is there a reason to redefine this action in this repro? For most other repos we import the template from Amplitude-TypeScript

E.g.
https://github.com/amplitude/Amplitude-Kotlin/blob/main/.github/workflows/jira-issue-create.yml

Reply:
Thanks for catching this.